core/trino-main/src/test/java/io/trino/execution/BaseDataDefinitionTaskTest.java

```diff
@@ -433,5 +455,18 @@ public ResolvedFunction getCoercion(Session session, OperatorType operatorType,
         {
             return delegate.getCoercion(session, operatorType, fromType, toType);
         }
+
+        private static ColumnMetadata withComment(ColumnMetadata tableColumn, Optional<String> comment)
+        {
+            return ColumnMetadata.builder()
```

Comment:
How about using `ColumnMetadata.builderFrom`? 
```java
            return ColumnMetadata.builderFrom(tableColumn)
                    .setComment(comment)
                    .build();
```

Reply:
I missed this utility method. Thanks for pointing it out.